game-core/src/main/java/games/strategy/engine/chat/ChatPlayerPanel.java

```diff
@@ -255,9 +253,6 @@ private String getDisplayString(final INode node) {
       }
       extra = extra + " (" + sb + ")";
     }
-    if (extra.length() == 0) {
-      return node.getName();
-    }
     return node.getName() + extra;
```

Comment:
Those length == 0 checks are so weird

Reply:
Yeah, this whole thing can be simplified quite a bit. The `getNotesForNode` method too, in the end should become more or less a simple concatenation